src/bci_build/package/git/README.md.j2

```diff
@@ -0,0 +1,36 @@
+# {{ image.title }}: Git application container
+{% include 'badges.j2' %}
+
+
+## Description
+
+Git is a distributed version control system that tracks
+versions of files. It is often used to control source code by programmers who
+are developing software collaboratively.
+
+Design goals of Git include speed, data integrity, and support for
+distributed, non-linear workflows — thousands of parallel branches running on
+different computers.
+
+
+## Usage
+
+This container provides the {{ image.publish_registry.vendor }} version of Git.
+
+Example of using Git container:
+
+```ShellSession
+$ podman run {{ image.pretty_reference }} git help
+```
+
+As Git requires a repository for most operations to function, the container
+does not explicitly set an entrypoint. This way you can launch the container in
+interactive mode to clone a repository and work on it. To have your changes persisted,
+you need to use a persistent volume mount on launch to avoid losing all your changes
+when exiting the container.
```

Comment:
```suggestion
As Git requires a repository, the container
does not explicitly set an entrypoint. This way, you can launch the container in
interactive mode to clone a repository and work on it. To avoid losing all your changes when exiting the container, use a persistent volume mount on launch.
```